### PR TITLE
Updated User Interface and Classes

### PR DESCRIPTION
**Pull request by manchava**

### CODE REVIEW

**Review by manchava — COMMENTED** [review]
I updated the main fragment and main activity with things that I felt were best suited for the purpose of the app. I added a search bar as well as constraints for users when they are looking for locations.

**Comment by manchava** [comment]
1) I believe that the LocationDAO class needs to be updated to contain information that will help call ArrayList holding the data needed for the app.
2) Are there enough dto files added to support all that is needed for the app?
3) Is the UI set up in the most efficient way and how will the different additions be included to navigate between the different pages?

**Comment by manchava** [comment]
The program was set up well. There were a lot of good classes included and set up well. There was a good start to the user interface as well. The dto was well implemented and there was good methods implemented throughout the Main activity and Main fragment.
The program was available in Github on time and I was added as a collaborator with ease.
There could have been more comments explaining the different things that were in different classes. Such as what the different methods do, what the ArrayLists are holding, and in which part of the app it will be used in. I think these comments would overall help get a better understanding of the function of the various lines of code.
Yes, the program compiles.
There was not a lot I felt needed changing, but I made a few additions. I added a little to the main page of the app that would help the user when adding places. I also added the start of the DAO interface to help set up for how it should begin.

**Comment by manchava** [comment]
1) My group was having trouble with the constraints for the different parts of the app and reviewing this code helped me get a better idea on how to implement it.

ex.  app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintHorizontal_bias="0.5"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent"

2) This code review helped me reinforce how to set up the dto's. Seeing how the group implemented a LocationDTO and UserDTO for the different informatin they would be collecting on the app helped me see its purpose and how to set it up.

ex. package uc.edu.ozturkkl.traveltracker.dto
data class Location(var address : String, var city : String, var state : String, var zip : Int) {
}

3) I learned what a companion object is while reviewing the code and how it is necessary in Kotlin as it does not have static members or member functions like java does, but it needed to access the information in this class.

ex. class MainFragment : Fragment() {
    companion object {
        fun newInstance() = MainFragment()
    }

File: app/src/main/java/uc/edu/ozturkkl/traveltracker/dao/LocationDAO.kt [hunk 1]
```diff
@@ -1,9 +1,18 @@
 package uc.edu.ozturkkl.traveltracker.dao
 
-import uc.edu.ozturkkl.traveltracker.dto.Location
+import android.telecom.Call
+
 
 interface LocationDAO {
 
 
+    //fun getAllLocations():
```
Comment (discospiff) [inline app/src/main/java/uc/edu/ozturkkl/traveltracker/dao/LocationDAO.kt:9]: Don't commit commented out code to version control.  That's a no-no.  

1. You are in your own branch, you can do what you want without impacting master.  That's why this review is a separate branch.
2. You can always go back in history to see what used to be there.
3. You should only commit compiling code to master.   
4. In the instructions, I mentioned that you should make proposals in code; not have a discussion in code.

**Comment by discospiff** [comment]
The write up is good, but I lean against merging this change.
1) Commented-out code does not belong in version control, much less master.
2) The layout changes are essentially a new, and partially complete feature.  Only complete features should be merged to master.  
Another option, if the group likes this proposal: continue working in this branch until the feature is complete, and then merge.

**Pull request closed without merging**